Update/kage_003.txt

```diff
@@ -4683,10 +4683,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500355", 256, TRUE);
 	OutputLine(NULL, "「要請を受けて保健所には連絡したんですが、…鳥だと食用の場合もありますから、注意くらいしかできないそうです。",
-			NULL, "\"I reported it to the public health center... but they said there wasn't much they could do, since the birds were edible animals.", Line_WaitForInput);
+			NULL, "\"I reported it to the public health center... but they said they couldn't do more but give them a warning, since the birds were edible animals.", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, "\"I reported it to the public health center... but they said they couldn't do more than give them a warning, since the birds were edible animals.", Line_WaitForInput);
```